test/Constraints/bridging_nonobjc.swift

```diff
@@ -1,5 +0,0 @@
-// RUN: %target-swift-frontend -typecheck -verify %s -disable-objc-interop
-
-
-var x: Any = 1
-var y = x as AnyObject // expected-error{{not convertible}}
```

Comment:
We should keep this test, but change the expected output.

Reply:
Ok!